### ISSUE TITLE
Training time?

### ISSUE THREAD

**Issue by hcguoO0** [OmidPoursaeed/Generative_Adversarial_Perturbations#5]
Hi, I try to run your code with a subset of Imagenet dataset about 8000 pictures to generate non-targeted imdep perturbation, but it takes so long time, about 10 hours. I wonder how long time do you spend to train such model ?

**Comment by OmidPoursaeed** [comment]
Hello, are you setting --MaxIter correctly (to run on a subset of the training set)? How many epochs are you using for training? Also, please make sure you are running it on GPU.

**Comment by hcguoO0** [comment]
Thank you for your answer, I have another question, is the  weights_init and the torch.log(the loss function) necessary?

**Comment by OmidPoursaeed** [comment]
Initializing the weights (weights_init) helps with better convergence. We found that using log helps, but it's not necessary.

**Comment by hcguoO0** [comment]
Thanks for your response.
